rosdep/python.yaml

```diff
@@ -10253,6 +10253,18 @@ python3-uvloop:
   nixos: [python3Packages.uvloop]
   openembedded: [python3-uvloop@meta-ros2]
   ubuntu: [python3-uvloop]
+python3-validators:
+  debian: [python3-validators]
+  fedora: [python3-validators]
+  opensuse: [python3-validators]
+  ubuntu:
+    '*': [python3-validators]
+    bionic:
+      pip:
+        packages: [validators]
+    focal:
+      pip:
+        packages: [validators]
```

Comment:
Bionic is un-supported so we shouldn't add handling for it. Focal is nearly EOL so I think it's not worth adding unless you're actually using it.

```suggestion
```